src/SSHDebugPS/UI/ViewModels/ContainerPickerViewModel.cs

```diff
@@ -160,9 +228,27 @@ private bool IsLibLinuxAvailable()
             }
         }
 
+        // Default is to Support SSH Connections
+        private bool _supportSSHConnections = true;
+        public bool SupportSSHConnections
+        {
+            get
+            {
+                return _supportSSHConnections;
+            }
+            private set
+            {
+                _supportSSHConnections = value;
+                OnPropertyChanged(nameof(SupportSSHConnections));
+            }
+        }
```

Comment:
Does this ever change or can it just be a read only property that is set in the constructor?

Reply:
this still builds against 7.3. I'm going to leave it as it is for now.